### ISSUE TITLE
Assimp exception when loading some vrm models

### ISSUE THREAD

**Issue by ltndkl** [ruyo/VRM4U#4]
Hello ruyo. I'm very interested in this plugin, but I can't import some models due to the **_DeadlyImportError_** exception (both in the release you provided and in my compilation of the plugin with your assimp).
This exception is only thrown for some vrm models like [this](https://3d.nicovideo.jp/works/td55734). I tested some different models, and the plugin works fine. The mentioned model causes problems in more than this plugin (I've also tried another plugin that can convert the VRM format, but the final results were weird), but can indeed be loaded using the VRMViewer.
The exception was from LazyDict<T>::Retrieve(unsigned int i) in glTF2Asset.inl (line 260):
```
 // read it from the JSON object
    if (!mDict) {
        throw DeadlyImportError("GLTF: Missing section \"" + std::string(mDictId) + "\"");
    }
```
where `mDictId` was _meshes_, `i` was _1_, 
`mObjs` was
![mObjs](https://user-images.githubusercontent.com/13776187/66720466-a031c280-ee2f-11e9-8e37-88dcb14cafdc.png)
`mObjsByOIndex` was
![mObjsByOIndex](https://user-images.githubusercontent.com/13776187/66720493-df601380-ee2f-11e9-9ee6-3e3339c9ab46.png)

The callstack was:
_glTF2Asset.inl_:

`glTF2::LazyDict<glTF2::Mesh>::Retriete` line 260
`glTF2::GLTF2VRMMetadata::Read` line 1554
`glTF2::Asset::Load` line 1814

_glTF2Importer.cpp_:

`glTF2Importer::CanRead` line 140

_Importer.cpp_:

`Importer::ReadFile` line 595
`Importer::ReadFileFromMemory` line 494

_LoaderBPFunctionLibrary.cpp_:

`ULoaderBPFunctionLibrary::GetVRMMeta` line 243

_VRM4UImporterFactory.cpp_:

`UVRM4UImporterFactory::FactoryCreateBinary` line 156

...(Engine functions)

Could you please try that model using the plugin? It would be perfect if this problem gets solved. 
Anyway, thanks for sharing such a wonderful plugin. ありがとうございます！

**Comment by ruyo** [comment]
Hi ltndkl.

I fixed it. Please try the latest VRM4U plugin.

This VRM file has strange mesh order.
I rearranged deserialize order.
If you have other VRM files to crash, I will test it.
Thanks!

![image](https://user-images.githubusercontent.com/6472835/66731393-2848b400-ee92-11e9-87a2-ca84a1b58b76.png)

**Comment by ltndkl** [comment]
> Hi ltndkl.
> 
> I fixed it. Please try the latest VRM4U plugin.
> 
> This VRM file has strange mesh order.
> I rearranged deserialize order.
> If you have other VRM files to crash, I will test it.
> Thanks!
> 
> ![image](https://user-images.githubusercontent.com/6472835/66731393-2848b400-ee92-11e9-87a2-ca84a1b58b76.png)

Thanks a lot for the fix, you're really a god!

**Comment by ltndkl** [comment]
Well, this model now has other issues.
When previewing (the shadow problems only seemed to happen in the level template "Default"):
![preview](https://user-images.githubusercontent.com/13776187/66735751-d86deb00-ee99-11e9-9674-176e9ce88782.png)
When in-game (the hair problem happens everywhere):
![ingame](https://user-images.githubusercontent.com/13776187/66735805-f89daa00-ee99-11e9-926e-50fde74707aa.png)

Using the _MToon Unlit_ material, and _MToonActor_ for the model.
If I use the default _SkeletalMeshActor_, I can't see the edge outline effects, but the hair gets rendered normally.

**Comment by ruyo** [comment]
Hi.
I fixed it. Please try VRM4U released 20191015.

Outline artifact on backface(hair) is fixed.
But there are some artifact. You make mtoon_OutlineWidth=0 in M_seihuku_outline and place as a new actor.
It caused by thin model like cloth has 2 side polygons.

BoundsScale=2 to show all shadow. 
This model has no physics collision on hand and foot. It is clipped.

Thank you for using this plugin!

![image](https://user-images.githubusercontent.com/6472835/66782224-2f160c00-ef10-11e9-9c9b-25d585a30391.png)

![image](https://user-images.githubusercontent.com/6472835/66782245-39380a80-ef10-11e9-8a3f-f8d027668798.png)

**Comment by ltndkl** [comment]
> Hi.
> I fixed it. Please try VRM4U released 20191015.
> 
> Outline artifact on backface(hair) is fixed.
> But there are some artifact. You make mtoon_OutlineWidth=0 in M_seihuku_outline and place as a new actor.
> It caused by thin model like cloth has 2 side polygons.
> 
> BoundsScale=2 to show all shadow.
> This model has no physics collision on hand and foot. It is clipped.
> 
> Thank you for using this plugin!
> 
> ![image](https://user-images.githubusercontent.com/6472835/66782224-2f160c00-ef10-11e9-9c9b-25d585a30391.png)
> 
> ![image](https://user-images.githubusercontent.com/6472835/66782245-39380a80-ef10-11e9-8a3f-f8d027668798.png)

本当にありがとうございます！
And I'd like to suggest hiding components that have turned off "Cast Shadows" from the shadowmap capturing. It's up to you whether this should be added. I'm now satisfied with the plugin. Thanks again for your good work.
